dev-packages/application-package/src/application-package.ts

```diff
@@ -220,6 +228,10 @@ export class ApplicationPackage {
         return this.ifBrowser(this.backendModules, this.backendElectronModules);
     }
 
+    get targetElectronMasterModules(): Map<string, string> {
+        return this.ifElectron(this.backendElectronMasterModules, new Map<string, string>());
+    }
```

Comment:
This method isn't necessary here, right? I can understand the `targetFrontendModules` and the `targetBackendModules` but not this one. I think it is sufficient to have the other method (`backendElectronMasterModules`), and if not electron, then return with an empty map. You do not have electron master modules in the browser, so no need to mimic a symmetric API. What do you think?

Reply:
You're right, I'll get rid of the method.